### ISSUE TITLE
onclick Margin Bottom

### ISSUE THREAD

**Issue by alextorres50** [orhanobut/dialogplus#143]
Hi
i add a `setMargins(0,0,0,marginBottom)`

but in this space (45dp) de onclicklistener and touch doesn't work

any solution?

**Comment by orhanobut** [comment]
Do you mean click on the overlay area below dialog?

**Comment by orcuns** [comment]
Yes @orhanobut , if we set margin from Bottom, it is not possible to click overlay area and dismiss the dialog. Any recommendations??

**Comment by orcuns** [comment]
```
((dialogPlus.holderView.parent as LinearLayout).parent as FrameLayout)
.setOnClickListener {
                dialogPlus.dismiss()
}
```
This is a nice workaround if you've set margin from bottom and wanted to dismiss the dialog on overlay click
